web/src/features/fireWeather/components/tables/NoonWxValueTables.tsx

```diff
@@ -29,26 +31,29 @@ const sharedColumns: Column[] = [
     id: 'temperature',
     label: 'Temp (°C)',
     align: 'right',
-    format: (value: number): string => value.toFixed(TEMPERATURE_VALUES_DECIMAL)
+    format: formatTemperature,
+    maxWidth: 70
   },
   {
     id: 'relative_humidity',
     label: 'RH (%)',
     align: 'right',
-    format: (value: number): number => Math.round(value)
+    format: formatRelativeHumidity,
+    maxWidth: 70
   },
   {
     id: 'wind_direction',
-    label: 'Wind Dir (10m)',
+    label: 'Wind Dir (10m) °',
```

Comment:
```suggestion
    label: 'Wind Dir (10m) (°)',
```

Reply:
I was irrationally irritated by the fact that this was the only unit of measure not placed inside `( )`